common/artifacts/00_artifacts.txt

```diff
@@ -2010,8 +2010,7 @@ sword_tier_2_new = {
 	martial = 1
 	combat_rating = 7
 	command_modifier = {
-		heavy_infantry = 0.05
-		actual_light_infantry = 0.10
+		actual_heavy_infantry = 0.10
```

Comment:
Artifact modifiers are still unadapted
Here you have replaced two modifiers with one

Reply:
Correct.